samples/boards/altera_max10/sysid/src/main.c

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2017 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <misc/printk.h>
+#include "altera_avalon_sysid.h"
```

Comment:
no such file?

Reply:
this sample is only applicable to altera_max10 board. somehow it is being built for qemu_x86 by shippable. I only added altera_max10 under platform_whitelist in sample.yaml so it should not build for other targets.

Reply:
@dbkinder now it's fine, shippable is passed.